rules/Proj1103.md

```diff
@@ -0,0 +1,41 @@
+﻿---
+parent: Other
+ancestor: MSBuild
+---
+
+# Proj1103: Coverlet requires Microsoft.NET.Test.Sdk 17.5.0 or higher
+Coverlet is a cross-platform code coverage framework for .NET, with support for
+line, branch and method coverage. It requires `Microsoft.NET.Test.Sdk` 17.5.0
```

Comment:
Does this version not depend on the coverlet version being used?

Reply:
I did not search for version requirements of older versions, to be honest. Probably some work to get all the data.

Reply:
The known-issues page recommends `17.7.0` and up, but these version recommendations only seem to be for the `coverlet.collector` package

Reply:
The known-issues page recommends `17.7.0` and up, but these version recommendations only seem to be fore the `coverlet.collector` package

Reply:
We can consider not to advise a version. The README states that 17.5 is the minimum version though.

Reply:
The readme specifies it's only for `coverlet.collector`, same with the version mentioned in the known issues pages: https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/KnownIssues.md#upgrade-coverletcollector-to-version--100